src/main/scala/Generator.scala

```diff
@@ -1,13 +1,7 @@
 package testchipip
 
-import chisel3.{Module, Driver}
-import freechips.rocketchip.config.{Parameters, Config}
-import freechips.rocketchip.util.{GeneratorApp, HasGeneratorUtilities, ParsedInputNames}
-import java.io.{File, FileWriter}
-import firrtl.annotations.JsonProtocol
+import freechips.rocketchip.config.Parameters
+import firrtl.options.{StageMain}
+import freechips.rocketchip.system.{RocketChipStage}
 
-object Generator extends GeneratorApp {
```

Comment:
This is already rebased on #78 ?

Reply:
Wait a minute. Should this go into `dev` or into `master`. Did we decide to ditch the `dev` branch on `testchipip`?